fonts.json

```diff
@@ -172,5 +172,125 @@
         "fontLower": ["@", "8", "(", "|)", "3", "/=", "6", "#", "!", "_|", "|<", "|_", "/V\\", "/\\/", "0", "|*", "Q", "?", "5", "7", "(_)", "\\/", "vv", "><", "Y", "2"],
         "fontUpper": ["@", "8", "(", "|)", "3", "/=", "6", "#", "!", "_|", "|<", "|_", "/V\\", "/\\/", "0", "|*", "Q", "?", "5", "7", "(_)", "\\/", "vv", "><", "Y", "2"],
         "fontDigits": ""
+    },
+    {
+        "fontName": "",
+        "fontLower": "ａｂｃｄｅｆｇｈｉｊｋｌｍｎｏｐｑｒｓｔｕｖｗｘｙｚ",
+        "fontUpper": "ΛＢＣＤΞＦＧＨＩＪＫＬＭＮ♢ＰＱＲＳＴＵＶＷＸＹＺ",
+        "fontDigits": "０１２３４５６７８９"
+    },
+    {
+        "fontName": "",
+        "fontLower": "αႦƈԃҽϝɠԋιʝƙʅɱɳσρϙɾʂƚυʋɯxყȥ",
+        "fontUpper": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": "ąცƈɖɛʄɠɧıʝƙƖɱŋơ℘զཞʂɬų۷ῳҳყʑ",
+        "fontUpper": "ąცƈɖɛʄɠɧıʝƙƖɱŋơ℘զཞʂɬų۷ῳҳყʑ",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": "ค๖¢໓ēfງhiวkl๓ຖ໐p๑rŞtนงຟxฯຊ",
+        "fontUpper": "ค๖¢໓ēfງhiวkl๓ຖ໐p๑rŞtนงຟxฯຊ",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": "åß¢Ðê£ghïjklmñðþqr§†µvwx¥z",
+        "fontUpper": "ÄßÇÐÈ£GHÌJKLMñÖþQR§†ÚVW×¥Z",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": "ΛBᄃDΣFGΉIJKᄂMПӨPQЯƧƬЦVЩXYZ",
+        "fontUpper": "ΛBᄃDΣFGΉIJKᄂMПӨPQЯƧƬЦVЩXYZ",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": "₳฿₵ĐɆ₣₲ⱧłJ₭Ⱡ₥₦Ø₱QⱤ₴₮ɄV₩ӾɎⱫ",
+        "fontUpper": "₳฿₵ĐɆ₣₲ⱧłJ₭Ⱡ₥₦Ø₱QⱤ₴₮ɄV₩ӾɎⱫ",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": "ﾑ乃ᄃり乇ｷムんﾉﾌズﾚﾶ刀のｱゐ尺丂ｲひ√Wﾒﾘ乙",
+        "fontUpper": "ﾑ乃ᄃり乇ｷムんﾉﾌズﾚﾶ刀のｱゐ尺丂ｲひ√Wﾒﾘ乙",
+        "fontDigits": "0123456789"
+    },
+    {
+        "fontName": "",
+        "fontLower": ["【a】","【b】", "【c】", "【d】","【e】","【f】","【g】","【h】","【i】","【j】","【k】","【l】","【m】","【n】","【o】","【p】","【q】","【r】","【s】","【t】","【u】","【v】","【w】","【x】","【y】","【z】"],
+        "fontUpper": ["【A】","【B】","【C】","【D】","【E】","【F】","【G】","【H】","【I】","【J】","【K】","【L】","【M】","【N】","【O】","【P】","【Q】","【R】","【S】","【T】","【U】","【V】","【W】","【X】","【Y】","【Z】"],
+        "fontDigits": ["【0】","【1】","【2】","【3】","【4】","【5】","【6】","【7】","【8】","【9】"]
+    },
+    {
+        "fontName": "",
+        "fontLower": ["『a』","『b』","『c』","『d』","『e』","『f』","『g』","『h』","『i』","『j』","『k』","『l』","『m』","『n』","『o』","『p』","『q』","『r』","『s』","『t』","『u』","『v』","『w』","『x』","『y』","『z』"],
+        "fontUpper": ["『A』","『B』","『C』","『D』","『E』","『F』","『G』","『H』","『I』","『J』","『K』","『L』","『M』","『N』","『O』","『P』","『Q』","『R』","『S』","『T』","『U』","『V』","『W』","『X』","『Y』","『Z』"],
+        "fontDigits": ["『0』'<'『1』","『2』","『3』","『4』","『5』","『6』","『7』","『8』","『9』"]
```

Comment:
Incorrect formatting, the numbers won't show up properly.

Reply:
![image](https://user-images.githubusercontent.com/56948680/192104228-50a01dd4-5a19-4aa0-b73e-d0c919446b2e.png)